.github/ISSUE_TEMPLATE/experimental_feature.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Experimental Feature Requirements
```

Comment:
What is the point of experimental feature? Do we need to fill it out before merging it? Before telling users about it?

What qualifies as a "feature" and just a change?

Reply:
We have previously had items that were explicitly called out as experimental features in the TOC. As examples, there was new functionality that the networking group wanted feedback on before pursuing too far. There was also functionality from the user experience group that they wanted feedback from -- for the UX group, I believe it was alternatives to `istioctl wait`.

Reply:
This template should be filled out before such features go out in an Istio release. If users want to compile the code themselves, I think it's ok to tell them about it first.

Reply:
I think the template should be filled out before we ask any users to try out the feature. Maybe we can make that explicit in the description.

Reply:
Added a description.

Experimental features allow Istio work groups to collect more information
on potential new functionality. This template should be completed users try out any new experimental features.